mtui/actions/downgrade.py

```diff
@@ -0,0 +1,32 @@
+from string import Template
+
+from ..messages import MissingDowngraderError
+from ..utils import DictWithInjections
+
+
+def zypper() -> dict[str, Template]:
+    list_command_template = r"""
+for p in $packages; do \
+zypper -n se -s --match-exact -t package $$p; \
+done \
+| grep -v "(System" \
+| grep ^[iv] \
+| sed "s, ,,g" \
+| awk -F "|" '{{ print $2,"=",$4 }}'
```

Comment:
What about this then, should that be also escaped?
```suggestion
| awk -F "|" '{{ print $$2,"=",$$4 }}'
```

Reply:
Not needed as `$123` isn't recognized as template because number cat be keyword

Reply:
Ah, ok then... to much auto-magic 😄